src/integrationtests/java/com/aws/greengrass/integrationtests/deployment/SubGroupDeploymentIntegrationTest.java

```diff
@@ -91,12 +109,17 @@ public class SubGroupDeploymentIntegrationTest extends BaseITCase {
     private MqttClient mqttClient;
 
     private Map<String, CountDownLatch> groupLatchMap;
+    private Map<String, Map<String, Object>> groupDeploymentStatus;
+    private Map<String, AtomicReference<FleetStatusDetails>> fleetStatusUpdates;
 
     @BeforeEach
     void before(ExtensionContext context) throws Exception {
         ignoreExceptionOfType(context, PackageDownloadException.class);
         ignoreExceptionOfType(context, SdkClientException.class);
+        ignoreExceptionOfType(context, DeploymentRejectedException.class);
```

Comment:
recommend only doing this ignore in the tests which need it, not in the beforeeach

Reply:
I would need to get the ExtensionContext somehow in the test to set this up .. not sure if that's possible. I will look for it.

Reply:
You just add it as a parameter in the test method, it will be given to you.